web-app/pages/certificate/[tokenId].tsx

```diff
@@ -6,61 +6,90 @@ https://www.linkedin.com/post-inspector/
 https://cards-dev.twitter.com/validator
 */
 
-import type { NextPage } from 'next';
-import { useRouter } from 'next/router';
+import type { GetServerSideProps, NextPage } from 'next';
+import Head from 'next/head';
 import Layout from '../../components/Layout';
 import { baseUrl } from '../../helpers/strings';
 
-function buildTwitterUrl(pngUrl: string) {
+const title = 'I got certified on the NEAR blockchain!';
+const description = 'View NEAR University certificates of any .near account';
+// const certificateUrl = 'https://weak-dog-93.loca.lt/certificate/103216412152497cb6c193162a27c49h';
+function buildTwitterUrl(certificateUrl: string) {
   // https://stevenwestmoreland.com/2018/07/creating-social-sharing-links-without-javascript.html
-  const text = 'I got certified on the NEAR blockchain!';
   const hashtags = ['NEAR', 'blockchain', 'NEARUniversity'];
   const via = 'NEARedu';
-  const url = encodeURI(`https://twitter.com/intent/tweet?text=${text}&url=${pngUrl}&hashtags=${hashtags.join(',')}&via=${via}`);
+  const url = encodeURI(`https://twitter.com/intent/tweet?text=${title}&url=${certificateUrl}&hashtags=${hashtags.join(',')}&via=${via}`);
   console.log({ url });
   return url;
 }
 
-function buildLinkedInUrl(pngUrl: string) {
+function buildLinkedInUrl(certificateUrl: string) {
   // https://stackoverflow.com/a/61583006/470749
-  const url = encodeURI(`https://www.linkedin.com/sharing/share-offsite/?url=${pngUrl}`);
+  const url = encodeURI(`https://www.linkedin.com/sharing/share-offsite/?url=${certificateUrl}&title=${title}&summary=${description}&source=LinkedIn`);
```

Comment:
I think `&source=LinkedIn` probably doesn't make sense and should be removed. Can you please make a note to yourself to remove it in a future branch (and test that it still works)?

Reply:
@rashaabdulrazzak Also, in response to https://github.com/NEAR-Edu/near-certification-tools/pull/20#issuecomment-1047917403, I personally have a slight preference for the second option. Which option is currently being used?

Reply:
the first option is currently used. I can change it to the second one easily